go/oasis-test-runner/scenario/e2e/runtime/txsource.go

```diff
@@ -148,11 +148,10 @@ func (sc *txSourceImpl) Fixture() (*oasis.NetworkFixture, error) {
 	f.Network.StakingGenesis = "tests/fixture-data/txsource/staking-genesis.json"
 
 	if sc.nodeRestartInterval > 0 {
-		// If node restarts enabled, do not enable round timeouts and merge
+		// If node restarts enabled, do not enable round timeouts and
```

Comment:
I'm not sure if `RoundFailures` are also expected to happen with runtime nodes getting restarted? (i guess will see after the test run i'm currently running will finish).

Reply:
This might have been there due to there not being enough validators (during a validator restart) which caused things to time out and fail? Maybe we fixed this with adding more validators, but I guess it can still happen in case we are unlucky and the proposer is the one being restarted (as this will cause things to fall over only after a timeout).

Reply:
so current change in this PR removes the `LogAssertNoExecutionDiscrepancyDetected()` - since this can now happen due to runtimes nodes being restarted.
 
And I am/was wondering if `RoundFailures` should also be removed, not sure if it can happen now?

Reply:
Based on the one full 6 hour run on this PR it looks like it doesn't happen, so the check can be kept?

Reply:
Yeah let's leave it in for now and we'll see if it causes problems.